webhook-integrations/Runscope/notes.md

```diff
@@ -0,0 +1,29 @@
+# Runscope Integration Investigation
+
+* [Runscope](https://runscope.com) is a company that provides solutions for API Performance Testing.
+
+* Runscope helps a company to conduct API Performance tests on an independent, hassle-free platform.
+
+* Runscope already has integrations with Slack & generates notifications in Slack for the following cases:
+
+  1. Test Passed
+
+  2. Test Failed
+
+* [The Notification Model Image](https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/notification_model.PNG)
+
+* [The 'Test Passed' notification](https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/test_passed.PNG)
+
+* [The 'Test Failed' notification](https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/test_failed.PNG)
+
+
+## Links
+
+Slack API - https://api.slack.com
+
+
+## Other Notes
+* Paid Subscription to Runscope enables users to create a larger team. It also increases test limits & increases
```

Comment:
Consider splitting this line at around 80 characters like you've split other lines.